TrackingTools/PatternTools/src/TempTrajectory.cc

```diff
@@ -118,6 +118,8 @@ bool TempTrajectory::badForCCC(const TrajectoryMeasurement &tm) {
     return false;
   if (thit->isPixel())
     return false;
+  if (!tm.updatedState().isValid())
```

Comment:
uhmmm, in principle we should NOT store invalid states in the trajectory. we should abort Pattern-reco for that traj...